e2e-tests/tests/03_update_appointment_attended_enforcement.spec.ts

```diff
@@ -24,7 +23,11 @@ test('Update a session appoinment with an attended but enforceable outcome', asy
   const attendanceOutcomePage = await completeCheckProjectDetails(page, checkProjectDetailsPage)
 
   const logHoursPage = await completeAttendedEnforceableOutcome(page, attendanceOutcomePage)
-  await completeHours(logHoursPage)
+
+  await logHoursPage.enterHours()
+  await logHoursPage.enterPenaltyHours()
```

Comment:
Are we ok with all routes entering penalty hours? As it's not required, we'd never catch any errors for if it wasn't provided. I think these are for happy path so probably ok, just a thought